ts/models/conversation.ts

```diff
@@ -1026,6 +1041,11 @@ export class ConversationModel extends Backbone.Model<ConversationAttributes> {
   public async addSingleMessage(messageAttributes: MessageAttributesOptionals, setToExpire = true) {
     const model = new MessageModel(messageAttributes);
 
+    const isMe = messageAttributes.source === UserUtils.getOurPubKeyStrFromCache();
+    if (isMe) {
+      await this.setIsApproved(true);
+    }
+
```

Comment:
you've got to not do this if the feature flag is off. and actually all calls to setIsApproved should not do anything